UITests/LandscapeToolbarTests.swift

```diff
@@ -0,0 +1,89 @@
+        UIDevice.currentDevice().setValue(value, forKey: "orientation")
+
+        tester().tapViewWithAccessibilityIdentifier("url")
+        var textView = tester().waitForViewWithAccessibilityLabel("Address and Search") as? UITextField
+        XCTAssertTrue(textView!.text.isEmpty, "Text is empty")
+        XCTAssertNotNil(textView!.placeholder, "Text view has a placeholder to show when it's empty")
+
+        // Navigate to two pages and press back once so that all buttons are enabled in landscape mode.
+        let url1 = "\(webRoot)/numberedPage.html?page=1"
+        tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url1)\n")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        tester().tapViewWithAccessibilityIdentifier("url")
+        textView = tester().waitForViewWithAccessibilityLabel("Address and Search") as? UITextField
+        XCTAssertEqual(textView!.text, url1, "Text is url")
+
+        let url2 = "\(webRoot)/numberedPage.html?page=2"
+        tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url2)\n")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 2")
+
+        tester().tapViewWithAccessibilityLabel("Back")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        // Open new tab and then go back to previous tab to test navigation buttons.
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        var addTabButton = tester().waitForViewWithAccessibilityLabel("Add Tab") as? UIButton
+        addTabButton?.tap()
+        tester().waitForViewWithAccessibilityLabel("Web content")
+
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        tester().tapViewWithAccessibilityLabel("Page 1")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        // Test to see if all the buttons are enabled.
+        tester().waitForTappableViewWithAccessibilityLabel("Back")
+        tester().waitForTappableViewWithAccessibilityLabel("Forward")
+        tester().waitForTappableViewWithAccessibilityLabel("Reload")
+        tester().waitForTappableViewWithAccessibilityLabel("Share")
+        tester().waitForTappableViewWithAccessibilityLabel("Bookmark")
+
+        // Go Back to other tab to see if all buttons are disabled.
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        tester().tapViewWithAccessibilityLabel("home")
+        tester().waitForViewWithAccessibilityLabel("Web content")
+
+        var back = tester().waitForViewWithAccessibilityLabel("Back") as? UIButton
+        var forward = tester().waitForViewWithAccessibilityLabel("Forward") as? UIButton
+        var reload = tester().waitForViewWithAccessibilityLabel("Reload") as? UIButton
+        var share = tester().waitForViewWithAccessibilityLabel("Share") as? UIButton
+        var bookmark = tester().waitForViewWithAccessibilityLabel("Bookmark") as? UIButton
```

Comment:
Nit: Just use `as!` everywhere here. It's often a bad idea to force-unwrap with `as!`, but unit tests are the exception -- a crash simply means a failure, and it doesn't affect the actual product.